addons/account/models/company.py

```diff
@@ -129,7 +126,17 @@ class ResCompany(models.Model):
     # Technical field to hide country specific fields in company form view
     country_code = fields.Char(related='country_id.code')
 
+    # Taxes
+    account_fiscal_country_id = fields.Many2one('res.country', string="Fiscal Country", compute='compute_account_tax_fiscal_country', store=True, readonly=False, help="The country to use the tax reports from for this company")
+    account_enabled_tax_country_ids = fields.Many2many(
+        string="l10n-used countries",
+        comodel_name='res.country',
+        compute='_compute_account_enabled_tax_country_ids',
+        help="Technical field containing the countries for which this company is using tax-related features"
+             "(hence the ones for which l10n modules need to show tax-related fields).")
```

Comment:
account_fiscal_country_id is an editable computed field. So, you want the user to be able to modify it. If the user changes the country_id of the company, we consider it's okay to recompute the account_fiscal_country_id, as it will be the same in 99.99999% percent of the cases.